db/go/20201008153033_update_jupyter_lab_template.go

```diff
@@ -34,6 +34,7 @@ containers:
           jupytertxt="/data/.jupexported.txt";
           if [ -f "$condayml" ]; then conda env update -f $condayml; fi;
           if [ -f "$jupytertxt" ]; then cat $jupytertxt | xargs -n 1 jupyter labextension install --no-build; fi;
+          if [ -f "$jupytertxt" ]; then jupyter lab build; fi;
```

Comment:
Why not `&&` this to the previous command? Wouldn't this still run even if the previous command wasn't successful?

```
if [ -f "$jupytertxt" ]; then cat $jupytertxt | xargs -n 1 jupyter labextension install --no-build && jupyter lab build; fi;
```

Reply:
Good point, change made.